firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/BreathingCircuit/AlarmMuteService.h

```diff
@@ -14,11 +14,23 @@
 
 namespace Pufferfish::Driver::BreathingCircuit {
 
-static const float countdown_time = 120;
+static const uint64_t countdown_time = 120;
```

Comment:
This should be named `max_mute_duration` or something similar. Also, we should use units of milliseconds for this constant, to be consistent with all other timestamps in the firmware.

Reply:
using `mute_max_duration `